README.md

```diff
@@ -105,31 +107,72 @@ Now, we use these predicted molecular substructures to generate a hierarchy of m
 2. A tree relating the MS1 features in these data based on molecular substructures predicted for MS1 features. This is of type `Phylogeny[Rooted]`. By default, we retain all fingerprint positions i.e. 2936 molecular properties). Adding `--p-qc-properties` filters these properties to keep only PubChem fingerprint positions (489 molecular properties) in the contingency table.
 **Note**: The latest release of [SIRIUS](https://www.nature.com/articles/s41592-019-0344-8) uses PubChem version downloaded on 13 August 2017.
-3. A combined feature data file that contains unique identifiers of each feature, their corresponding original feature identifier, and feature tables that each feature was detected in. This is of type `FeatureData[Molecules]`. (The renaming of features needs to be done to avoid overlapping, non-unique feature identifiers in the original feature table)
+3. A combined feature data file that contains unique identifiers of each feature, their corresponding original feature identifier (row ID in Mzmine2), CSI:FingerID structure predictions (SMILES), and the table(s) that each feature was detected in. This is of type `FeatureData[Molecules]`. (The renaming of features needs to be done to avoid overlapping, non-unique feature identifiers in the original feature table)
+
+These can be used as inputs to perform chemical phylogeny-based [alpha-diversity](https://docs.qiime2.org/2019.1/plugins/available/diversity/alpha-phylogenetic/) and [beta-diversity](https://docs.qiime2.org/2019.1/plugins/available/diversity/beta-phylogenetic/) analyses.
+
+Furthermore, Qemistree supports categorization of molecules into chemical taxonomy using [Classyfire](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0174-y) as follows:
 
+```bash
+qiime qemistree get-classyfire-taxonomy \
+  --i-feature-data merged-feature-data.qza
+  --o-classified-feature-data classified-merged-feature-data.qza
+```
+
+The resulting table is also of the type `FeatureData[Molecules]` includes categorization of molecules in the Classyfire levels 'kingdom', 'superclass', 'class', 'subclass', and 'direct_parent'.  
+Lastly, we include some utility functions in Qemistree that are most useful for visualizing the molecular hierarchy generated above.
+
+1. Prune molecular hierarchy to keep only the molecules with annotations.
+
+```bash
+qiime qemistree prune-hierarchy \
+  --i-feature-data classified-merged-feature-data.qza
+  --p-column subclass
+  --o-tree merged-qemistree.qza
+```
+
+Users can choose one of the following data columns (`--p-column`) for pruning: 'kingdom', 'superclass', 'class', 'subclass', 'direct_parent', and 'smiles'. All features with no data in this column will be removed from the phylogeny.
+
+2. Generate supporting files to visualize hierarchy using the web-based service [iTOL](https://itol.embl.de/).
+
+```bash
+python ../_itol_metadata.py \

```

Comment:
The path to this script is going to be installation dependent. Alternatively we could suggest that people run this from within this repository. Any ideas?

Reply:
Great point, I've made a note of this in the new update. Let me know if it would suffice!